official/recommendation/ncf_keras_benchmark.py

```diff
@@ -213,3 +213,50 @@ def benchmark_2_gpus(self):
     self._setup()
     FLAGS.num_gpus = 2
     self._run_and_report_benchmark()
+
+class NCFKerasPerf(NCFKerasBenchmarkBase):
```

Comment:
r/NCFKerasPerf/NCFKerasBenchmark

NCF "breaks" our pattern in that "Benchmark" is often just a few steps.  For ResNet50 I put the "mlperf like" tests under accuracy.  You could also choose to do that and not have this Class at all.  My  guideline, for myself, is if the test is "long" it falls under accuracy; which is a false assumption I choose to ignore for now.  If the test is testing examples/sec and short (a few steps) then it is a benchmark test.  What you pick is fine with me as it is a grey area that is my fault.